db/migrate/20171121110841_create_link_histories.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLinkHistories < ActiveRecord::Migration[5.0]
+  def change
+    create_table :link_histories do |t|
+      t.json :link_errors, default: []
+      t.belongs_to :link
```

Comment:
Can we use `t.references :link, index: true, foreign_key: true` to get the foreign key as well?